analyzers/tests/SonarAnalyzer.UnitTest/TestCases/LdapConnectionShouldBeSecure.CSharp9.cs

```diff
@@ -8,12 +8,12 @@
 new DirectoryEntry("path", "user", "pass", authTypeSecure); // Compliant
 
 var authTypeNone = AuthenticationTypes.None;
-new DirectoryEntry("path", "user", "pass", authTypeNone);   // FN
+new DirectoryEntry("path", "user", "pass", authTypeNone);   // Noncompliant
```

Comment:
maybe you can remove these rules (Ldap, jwt) from the top-level support MMF?

Reply:
None of these (S4433, S5659) is part of the [top-level support MMF](https://jira.sonarsource.com/browse/MMF-2222)